src/table/sorting-model.ts

```diff
@@ -0,0 +1,27 @@
+export class SortingModel {
+    key: string;
+    direction: string = 'original';
+    ascCallback: (a, b) => (number);
+    descCallback: (a, b) => (number);
+    constructor() {
+        this.ascCallback = function (a, b) {
+            if (a[this.sortingModel.key] > b[this.sortingModel.key]) {
+                return 1;
+            }
+            if (a[this.sortingModel.key] < b[this.sortingModel.key]) {
+                return -1;
+            }
+            return 0;
+        };
+
+        this.descCallback = function (a, b) {
```

Comment:
I feel that these callbacks are redundant and it's possible to improve it if you will create `sortFunc` in utils or somewhere else. Possible implementation:  
```typescript
export function sortFunc(prop: string, order: SortDirection) {
    if (!prop) return 0;
    return (a, b) => {
        if (!a.hasOwnProperty(prop) || !b.hasOwnProperty(prop)) return 0;
        let comparison = 0;
        if (a[prop] > b[prop]) comparison = 1;
        if (a[prop] < b[prop]) comparison = -1;
        return order === 'desc' ? comparison * -1 : comparison;
    }
}
```